deployments/goerli/weth/deploy.ts

```diff
@@ -0,0 +1,148 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { debug, DeploySpec, deployComet, exp, sameAddress, wait } from '../../../src/deploy';
+
+const clone = {
+  cbETHImpl: '0x31724cA0C982A31fbb5C57f4217AB585271fc9a5',
+  cbETHProxy: '0xBe9895146f7AF43049ca1c1AE358B0541Ea49704',
+};
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const deployed = await deployContracts(deploymentManager, deploySpec);
+  await mintTokens(deploymentManager);
+  return deployed;
+}
+
+async function deployContracts(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const ethers = deploymentManager.hre.ethers;
+  const signer = await deploymentManager.getSigner();
+
+  // Declare existing assets as aliases
+  const WETH = await deploymentManager.existing('WETH', '0x42a71137C09AE83D8d05974960fd607d40033499', 'goerli');
```

Comment:
looks like there are a number of different instances of the `WETH9` contract on Goerli.

Looks like Uniswap uses `0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6`: https://docs.uniswap.org/contracts/v3/reference/deployments

Reply:
Yeah, for some reason our Goerli USDC deployment has its version of WETH. I chose to use that same WETH to be consistent.